lib/click.ts

```diff
@@ -40,6 +37,61 @@ export function click(params: InsightsSearchClickEvent) {
     );
   }
 
-  // Send event
-  this.sendEvent("click", params);
+  this.sendEvent("click", params as InsightsEvent);
+}
+
+export interface InsightsClickObjectIDsEvent {
+  eventName: string;
+  userID: string;
+  timestamp: number;
```

Comment:
yes, it will always exist.
The only thing missing is that it need to be optional. I'll make a PR for this specifically. Open an issue: https://github.com/algolia/search-insights.js/issues/new